services/movie.py

```diff
@@ -0,0 +1,33 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids=None, actors_ids=None):
+    queryset = Movie.objects.all()
+    if genres_ids is None and actors_ids is None:
+        return queryset
+    if genres_ids is not None and actors_ids is not None:
+        queryset = queryset.filter(genres__id__in=genres_ids,
+                                   actors__id__in=actors_ids)
+    if genres_ids is not None and actors_ids is None:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids is not None and genres_ids is None:
+        queryset = queryset.filter(actors__id__in=actors_ids)

```

Comment:
It is a bad approach. You can simply use `if genres_ids` and `if actors_ids` to filter the previous queryset.